### ISSUE TITLE
After deploying dev branch sometimes JS error accepting changes

### ISSUE THREAD

**Issue by tuhlmann** [loopindex/ckeditor-track-changes#82]
We ran into an issue where in one instance we receive a bunch JS errors when trying to accept all pending changes.

Here's the error:

```
TypeError: Cannot read property 'attributes' of null
    at assets-0.1.0.min.78672032.js:39
    at Object.window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.htmlParser.fragment.forEach (assets-0.1.0.min.78672032.js:27)
    at Object.window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.htmlParser.fragment.forEach (assets-0.1.0.min.78672032.js:27)
    at Object.window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.htmlParser.fragment.forEach (assets-0.1.0.min.78672032.js:27)
    at Object.window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.htmlParser.fragment.forEach (assets-0.1.0.min.78672032.js:27)
    at Object.window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.htmlParser.fragment.forEach (assets-0.1.0.min.78672032.js:27)
    at null.<anonymous> assets-0.1.0.min.78672032.js:39)
    at g (assets-0.1.0.min.78672032.js:23)
    at window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.event.CKEDITOR.event.fire (assets-0.1.0.min.78672032.js:23)
    at window.CKEDITOR.window.CKEDITOR.dom.CKEDITOR.editor.CKEDITOR.editor.fire (assets-0.1.0.min.78672032.js:23)
```

The code (minified) that this points to is:

```
e=a.instances[e["data-cke-widget-id"]])&&(d=c.getFirst(b.isParserWidgetElement),f.push({wrapper:c,element:d,widget:e,editables:{}}),"1"!=d.attributes["data-cke-widget-keep-attr"]&&delete d.attributes["data-widget"])
```

I'm not sure if this is related to the track changes plugin, it just resurfaced after I updated it.

The other exception I've seen is:

```
Uncaught ReferenceError: LITE_LABELS_DATE is not defined

a	@	plugin.js?t=F7J9:2
f._makeTooltipTitle	@	plugin.js?t=F7J9:2
f._showTooltip	@	plugin.js?t=F7J9:2
(anonymous function)	@	lite-includes.js:2
```

Thanks,
Torsten.

**Comment by imdfl** [comment]
Regarding the exception, I'd be happy to help if you provide a reproducible scenario.
The global LITE_LABELS_DATE has been removed in both dev and master. Sorry bout that.

**Comment by tuhlmann** [comment]
Thanks. I didn't know we access `LITE_LABELS_DATE`, I assumed that is some dev branch error...

Concerning the other error, I upgraded ckeditor to 4.5.5 and tested with the lite plugin from master branch and with the one we had installed previously. Turns out the error appears in all of them, so I guess it's a problem of this particular set of data.

Interestingly this error appears twice with the dev branch and only once with the master branch and the old version.

I close this ticket.
